variables.pkr.hcl

```diff
@@ -64,13 +64,13 @@ variable "containerd_version" {
 variable "docker_version_al2022" {
```

Comment:
now that docker_version_al2022 is in sync with AL2 docker version, we could completely remove this variable and use [docker_version](https://github.com/aws/amazon-ecs-ami/blob/main/release.auto.pkrvars.hcl#L4) in all places for AL2022 ([example](https://github.com/aws/amazon-ecs-ami/blob/main/al2022.pkr.hcl#L28)).

Same applies to [containerd_version_al2022](https://github.com/aws/amazon-ecs-ami/blob/179903b83d50a04393b5b231ba9b9954dcb6cde9/variables.pkr.hcl#L70)

Reply:
Since AL2022 is still in preview, I keep these two variables for AL2022 just in case we need another docker upgrade for ECS optimized AL2 AMI and its variants, but new docker version is not available for AL2022 at that time. We can remove them when AL2022 is GA : )